docs/tutorials/geomesa-quickstart-fsds.rst

```diff
@@ -0,0 +1,256 @@
+download and install version |geoserver_version|. Then follow the instructions in :ref:`install_fsds_geoserver`
+to enable GeoMesa.
+
+Register the GeoMesa Store with GeoServer
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Log into GeoServer using your user and password credentials. Click
+"Stores" and "Add new Store". Select the ``FileSystem (GeoMesa)`` vector data
+source, and fill in the required parameters.
+
+Basic store info:
+
+-  ``workspace`` this is dependent upon your GeoServer installation
+-  ``data source name`` pick a sensible name, such as ``geomesa_quick_start``
+-  ``description`` this is strictly decorative; ``GeoMesa quick start``
+
+Connection parameters:
+
+-  these are the same parameter values that you supplied on the
+   command-line when you ran the tutorial; they describe how to connect
+   to the FSDS instance where your data reside
+
+Click "Save", and GeoServer will search your FSDS directory for any
+GeoMesa-managed feature types.
+
+Publish the Layer
+~~~~~~~~~~~~~~~~~
+
+GeoServer should recognize the ``gdelt-quickstart`` feature type, and
+should present that as a layer that can be published. Click on the
+"Publish" link.
+
+You will be taken to the Edit Layer screen. You will need to enter values for the data bounding
+boxes. In this case, you can click on the link to compute these values from the data.
+
+Click on the "Save" button when you are done.
+
+Take a Look
+~~~~~~~~~~~
+
+Click on the "Layer Preview" link in the left-hand gutter. If you don't
+see the quick-start layer on the first page of results, enter the name
+of the layer you just created into the search box, and press
+``<Enter>``.
+
+Once you see your layer, click on the "OpenLayers" link, which will open
+a new tab. You should see a collection of red dots similar to the following image:
+
+.. figure:: _static/geomesa-quickstart-accumulo/geoserver-layer-preview.png
+    :alt: Visualizing quick-start data
+
+    Visualizing quick-start data
+
+Tweaking the display
+~~~~~~~~~~~~~~~~~~~~
+
+Here are just a few simple ways you can play with the visualization:
+
+-  Click on one of the red points in the display, and GeoServer will
+   report the detail records underneath the map area.
+-  Shift-click to highlight a region within the map that you would like
+   to zoom into.
+-  Click on the "Toggle options toolbar" icon in the upper-left corner
+   of the preview window. The right-hand side of the screen will include
+   a "Filter" text box. Enter ``EventCode = '051'``, and press on the
+   "play" icon. The display will now show only those points matching
+   your filter criterion. This is a CQL filter, which can be constructed
+   in various ways to query your data. You can find more information
+   about CQL from `GeoServer's CQL
+   tutorial <http://docs.geoserver.org/2.9.1/user/tutorials/cql/cql_tutorial.html>`__.
+
+Generating Heatmaps
+~~~~~~~~~~~~~~~~~~~
+
+-  To try out server-side processing, you can install the Heatmap SLD from
```

Comment:
did you try this out? i don't think it will work with the fsds - you have to use the non-distributed geoserver heatmap prorcess. we should probably just remove this section